templates/ssl.crt.j2

```diff
@@ -0,0 +1 @@
+{{ ssl_certs_crt }}{{ digicert_crt }}
```

Comment:
There are two more variables here. Need to document. Better use the same naming conventions for them as well.

These variables contain the actual certificate content. Maybe the path variables should end with _path.